.github/workflows/test.yaml

```
@@ -0,0 +1,46 @@
+        uses: aa900031/github-actions/setup-node@main
+        with:
+          no-turbo-cache: 'true'
+
+      - name: Run Prepare
+        run: pnpm run dev:prepare
+
+      - name: Run Test
+        run: pnpm run test --coverage
+
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.8)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)